use-subscription/index.js

```diff
@@ -48,14 +49,25 @@ function useSubscription (channels, opts = {}) {
     .join(' ')
 
   useEffect(() => {
+    let timeout
     let ignoreResponce = false
-    changeSubscribing(true)
+
+    if (debounce > 0) {
+      timeout = setTimeout(() => {
+        changeSubscribing(true)
```

Comment:
Will it set `isSubscribing=false` on initial loading?

Reply:
Yes: https://github.com/euaaaio/redux/blob/feat/use-subscription-debounce/use-subscription/index.test.ts#L286